lib/LambdaLog.js

```diff
@@ -58,7 +58,7 @@ class LambdaLog extends EventEmitter {
             dynamicMeta: null,
             debug: false,
             dev: false,
-            silent: false,
+            silent: typeof process.env.LAMBDALOG_SILENT !== 'undefined',
```

Comment:
There would need to be multiple checks here as this should account for `LAMBDALOG_SILENT=false` as well.